### ISSUE TITLE
Consider handling transformations

### ISSUE THREAD

**Issue by grovesNL** [grovesNL/glyphon#4]
Consider handling scale, shear, rotation, etc., at least for typical 2D use cases

Also related to #3 because pixel snapping wouldn't work well depending on the transformation.

**Comment by mfluehr** [comment]
Personally, I mostly care about translation.

I'm building a 2D game that has a moving camera. I would like a way to apply a single camera to all `TextArea`s within a given `TextRenderer`. Currently, I have to manually update every single `TextArea` whenever the camera moves, which is inconvenient. I'd much rather leave them unchanged, and apply a single shader matrix to move everything at once.

To summarize, it seems there are two separate needs for this issue:
1. Transform individual `TextArea`s, as in #64.
2. Transform `TextRenderer`s, as in my use case.

**Comment by grovesNL** [comment]
Do you just want a convenient way to apply translation across all text areas, or does it need to happen in the shader itself? I'm just curious because we currently rely on having the coordinates on the CPU so we can clip/cull before it reaches the shader.

**Comment by mfluehr** [comment]
The advantage of GPU-side is that only the camera values need to change and all the text areas can stay the same. But your point about culling CPU-side is a good one, so I'd be fine applying my transforms prior to hitting the shader.
